chrome_manifest_v3/views/popupView.js

```
@@ -423,14 +425,14 @@ Gmail2Trello.PopupView.prototype.bindEvents = function () {
 
   var $board = $('#g2tBoard', this.$popup);
   $board.off('change').on('change', () => {
-    var boardId = $board.val();
+    let boardId = $board.val();
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'boardId' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)